<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.1870" tests="3" failures="1" skipped="2">
  <testsuite name="Root Suite" timestamp="2023-03-07T08:16:11" tests="0" file="cypress/e2e/linkme.cy.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="linkme" timestamp="2023-03-07T08:16:11" tests="3" time="0.1870" failures="1">
    <testcase name="linkme redirects on no rec.id" time="0.1870" classname="redirects on no rec.id">
      <failure message="`cy.visit()` failed trying to load:

http://localhost:3000/linkme.php/?isBot=true

The response we received from your web server was:

  &gt; 500: Internal Server Error

This was considered a failure because the status code was not `2xx`.

This http request was redirected &apos;1&apos; time to:

  - 308: http://localhost:3000/linkme.php?isBot=true

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

http://localhost:3000/linkme.php/?isBot=true

The response we received from your web server was:

  > 500: Internal Server Error

This was considered a failure because the status code was not `2xx`.

This http request was redirected '1' time to:

  - 308: http://localhost:3000/linkme.php?isBot=true

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`
    at http://localhost:4000/__cypress/runner/cypress_runner.js:156624:84
    at visitFailedByErr (http://localhost:4000/__cypress/runner/cypress_runner.js:155945:12)
    at http://localhost:4000/__cypress/runner/cypress_runner.js:156604:13
    at tryCatcher (http://localhost:4000/__cypress/runner/cypress_runner.js:13012:23)
    at Promise._settlePromiseFromHandler (http://localhost:4000/__cypress/runner/cypress_runner.js:10947:31)
    at Promise._settlePromise (http://localhost:4000/__cypress/runner/cypress_runner.js:11004:18)
    at Promise._settlePromise0 (http://localhost:4000/__cypress/runner/cypress_runner.js:11049:10)
    at Promise._settlePromises (http://localhost:4000/__cypress/runner/cypress_runner.js:11125:18)
    at _drainQueueStep (http://localhost:4000/__cypress/runner/cypress_runner.js:7719:12)
    at _drainQueue (http://localhost:4000/__cypress/runner/cypress_runner.js:7712:9)
    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:4000/__cypress/runner/cypress_runner.js:7728:5)
    at Async.drainQueues (http://localhost:4000/__cypress/runner/cypress_runner.js:7598:14)
From Your Spec Code:
    at Context.eval (http://localhost:4000/__cypress/tests?p=cypress/e2e/linkme.cy.js:114:8)]]></failure>
    </testcase>
  </testsuite>
</testsuites>